tools/gfx/renderer-shared.h

```diff
@@ -1186,14 +1187,56 @@ class TransientResourceHeapBase : public ITransientResourceHeap, public Slang::C
     }
 public:
     SLANG_COM_OBJECT_IUNKNOWN_ALL
-        ITransientResourceHeap* getInterface(const Slang::Guid& guid)
+    ITransientResourceHeap* getInterface(const Slang::Guid& guid)
     {
         if (guid == GfxGUID::IID_ISlangUnknown || guid == GfxGUID::IID_ITransientResourceHeap)
             return static_cast<ITransientResourceHeap*>(this);
         return nullptr;
     }
 };
 
+class ShaderTableBase
+    : public IShaderTable
+    , public Slang::ComObject
+{
+public:
+    Slang::List<Slang::String> m_entryPointNames;
```

Comment:
Perhaps benefit for comment to say the names are in the ordered contiguously over rayGenShaderCount, missShader etc.